src/pages/MetricDetail.svelte

```diff
@@ -36,23 +37,28 @@
 
   import { isExpired, isRemoved, isRecent } from "../state/items";
 
+  import { getGleanQuery, getGleanEventQuery } from "../data/gleanSql";
+
   export let params;
 
   let selectedAppVariant;
   let selectedPingVariant;
   let pingData = {};
 
-  const STMO_GLEAN_QUERY_URL =
-    "https://sql.telemetry.mozilla.org/queries/91729/source";
-  const STMO_GLEAN_EVENT_QUERY_URL =
-    "https://sql.telemetry.mozilla.org/queries/91780/source";
```

Comment:
we don't need templatized Redash queries since pre-generated queries are now available.